packages/flutter/test/material/toggle_buttons_test.dart

```diff
@@ -250,6 +250,74 @@ void main() {
     },
   );
 
+  testWidgets('Default text style is applied', (WidgetTester tester) async {
+    final ThemeData theme = ThemeData();
+    await tester.pumpWidget(
+      Material(
+        child: boilerplate(
+          child: ToggleButtons(
+            isSelected: const <bool>[false, true],
+            onPressed: (int index) {},
+            children: const <Widget>[
+              Text('First child'),
+              Text('Second child'),
+            ],
+          ),
+        ),
+      ),
+    );
+
+    TextStyle textStyle;
+    textStyle = tester.firstWidget<DefaultTextStyle>(
```

Comment:
Is "firstWidget" really needed here (and elsewhere)?

Reply:
Yes -- There are two `Material` widgets in this test -- The one explicitly instantiated and another one by `RawMaterialButton`, which is used to create each toggle button.

Reply:
Discussed offline and updated these finders to be more specific and easier to read